### DISCUSSION TITLE
Small intestinal transit time.

### DISCUSSION THREAD

**Discussion by JoseDavidGomez** [Open-Systems-Pharmacology/Forum#750] [Questions & Problems]
Dear community,

I have been exploring the effect of reduced intestinal transit time on drug absorption. In concrete the effect of excipients on transit time and absorption. 
The Pk-Sim default value for small intestinal transit time is 126 mins,  However in this publication:
https://link.springer.com/article/10.1023/A:1016256619309

they report median transit times of 242 mins. Does somebody know the PK-Sim reference for the default value. 

Thank you!

JD

**Comment by jzq90** [comment]
I found several publications about intestinal transit in the OPS instructions. Below is one of them, in which it reviewed the results and translate the results into PK-Sim. Even though I don't find where the 126 min come from, it may be helpful to you.
https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1002%2Fjps.22726

**Comment by Yuri05** [comment]
>I found several publications about intestinal transit in the OPS instructions. Below is one of them, in which it reviewed the results and translate the results into PK-Sim. Even though I don't find where the 126 min come from, it may be helpful to you.
https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1002%2Fjps.22726

Yep, this is the correct reference. Literature data from several sources was approximated by the sigmoidal function as shown in the paper. Small intestinal transit time was calculated then based on the definition below.

_Small intestinal transit time: the time required for the passage of a non-absorbable compound through the small intestine, defined as the **time span between gastric emptying** (i.e. the point at which 63% of the drug mass has entered the duodenum) **and the time at which 90% has entered the caecum**._

As you can see from the figure in the paper, 90% of dissolved drug pass the small intestine at around **2.35h**, giving (based on the default gastric emptying time = **15min**) small intestinal transit time = 2.35h - 0.25h = **2.1h** = **126min**

**Comment by StephanSchaller** [comment]
> ...approximated by the sigmoidal function as shown in the paper.

The paper does not describe how the (simulated) function is fitted (at least I could not find it), i.e., which parameters are fitted and what the simulated output actually represents.

The data shows e.g. the cumulated distribution of transit times across a population of 315 individuals (Data: Kim et al). Is this what the simulated curve also shows (with an underlying population simulation)? Or is it a single (mean representative) simulation showing the amount of a single administration reaching the caecum over time?

In the paper by Thelen et al. it states, that "(Fig. 1) According to this (sigmodial) function, transit through the small
intestine is almost complete after 3–4 h.". But the data from Kim et al. has a slightly different meaning and states, that 100% of individuals had transit times between 15 min and 4-5h.

While the outcome for the fitted function would be the same, it still does not explain how P1 and P2 from the SITT_factor and the TransitVelocity (or rate) for each segment is fitted/derived?
